### PR TITLE
feat: ui support for scheduling pdf exports

### PR DESCRIPTION
**Pull request by kandl**
Add support for scheduling pdf exports in the dashboard component. Refactor components to new model and methods.
Cleanup unnecessary bear-backend related logic.

risk: low
JIRA: F1-437

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

File: libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/hooks/useEditScheduledEmail.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+}: {
+    dashboardId: string;
+    dashboardTitle: string;
+    webhook: string;
+}): IAutomationMetadataObjectDefinition {
+    const firstRun = parseISO(new Date().toISOString());
+    const normalizedFirstRun = normalizeTime(firstRun);
+    const hour = normalizedFirstRun.getHours();
+    const minute = normalizedFirstRun.getMinutes();
+    const cron = `0 ${minute} ${hour} * * *`;
```
Comment (scavnickyj) [inline libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/hooks/useEditScheduledEmail.ts:161]: `0 ${minute} ${hour} * ? *`

File: libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/utils/repeat.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// (C) 2019-2023 GoodData Corporation
+// (C) 2019-2024 GoodData Corporation
```
Comment (scavnickyj) [inline libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/utils/repeat.ts:1]: This file can be removed right?

Comment (scavnickyj) [reply libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/utils/repeat.ts:1]: Maybe even some constants may be deleted?

File: libs/sdk-ui-dashboard/styles/scss/scheduled_mail.scss [hunk 3]
```diff
@@ -76,7 +83,7 @@ $vertical-space: 10px;
 
         .gd-input {
             flex-grow: 1;
-            max-width: 480px;
+            // max-width: 490px;
```
Comment (scavnickyj) [inline libs/sdk-ui-dashboard/styles/scss/scheduled_mail.scss:86]: remove

File: libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/DefaultScheduledEmailManagementDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        (scheduledEmail: IAutomationMetadataObject) => {
+            onEdit?.(scheduledEmail);
+        },
+        [onEdit],
+    );
+
+    const handleScheduleDeleteSuccess = useCallback(() => {
+        onDelete?.();
+        setScheduledEmailToDelete(null);
+        // TODO-MKN: refetch
```
Comment (scavnickyj) [inline libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/DefaultScheduledEmailManagementDialog.tsx:55]: What about the todo?

File: libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/hooks/useEditScheduledEmail.ts (hunk 1 above)
Comment (scavnickyj) [reply libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/hooks/useEditScheduledEmail.ts:161]: wrong string, sorry
`0 ${minute} ${hour} ? * *`

**Comment by kandl** [comment]
Closing in favor of https://github.com/gooddata/gooddata-ui-sdk/pull/5064

**Pull request closed without merging**